### DISCUSSION TITLE
关于本站

### DISCUSSION THREAD

**Discussion by giscus** [Xzonn/xzonn.github.io#6] [General]
# 关于本站

关于本站 - 来自Xzonn的小站

https://blog.www.localhost/posts/About.html

**Comment by Xzonn** [comment]
![icon_32x32](https://github.com/Xzonn/xzonn.github.io/assets/30619816/76ac68f5-ea96-46e6-8c28-7e5b03bf061a)

Hello, world!

**Comment by Xzonn** [comment]
测试。

**Reply by Xzonn** [reply]
测试回复 1。

**Reply by Xzonn** [reply]
测试图片：

![456](https://github.com/Xzonn/xzonn.github.io/assets/30619816/e88b8645-c758-463d-8cd4-a57ff3a9d174)

**Reply by XzonnWeblate** [reply]
你好！

**Comment by XzonnWeblate** [comment]
会有通知吗？
